corehq/apps/users/views/__init__.py

```diff
@@ -1004,3 +1004,12 @@ def location_restriction_for_users(request, domain):
         project.location_restriction_for_users = json.loads(request.POST["restrict_users"])
     project.save()
     return HttpResponse()
+
+
+@require_POST
+@require_superuser
+def register_fcm_device_token(request, couch_user_id, device_token):
```

Comment:
@sjain28 I mean any other HQ APIs that CommCare hits which don't have an implied application.